crates/containerd-shim-wasm/src/sandbox/oci.rs

```diff
@@ -36,6 +36,25 @@ pub fn get_args(spec: &Spec) -> &[String] {
     }
 }
 
+pub fn get_module(spec: &Spec) -> (Option<String>, String) {
```

Comment:
Could you please write docs for the return types of this method?

Reply:
done